neo/sys/sys_imgui.h

```diff
@@ -17,6 +15,8 @@ enum D3ImGuiWindow {
 
 #ifndef IMGUI_DISABLE
 
+#include "../libs/imgui/imgui.h"
```

Comment:
you're moving the include into `namespace D3::ImGuiHook` - that can't work, of course.